src/gp_doctest.cpp

```diff
@@ -1989,3 +1987,128 @@ TEST_CASE("Top-Pruning: Likelihoods") {
       fasta_path_six, newick_path_six_simple, false, false);
   CHECK_MESSAGE(test_3, "Six Taxa Multi Tree failed.");
 }
+
+// Runs an instance of TPEngine for two DAGs: DAG_1, a simple DAG, and DAG_2, a DAG
```

Comment:
This is a great overall summary but it would be 🤩 if it was easier to connect it to the actual test below. Could you use the variable names here or change the variable names below to match? For example, I'm kind of lost what `new_likelihoods_without_nni` is in this description without reading everything in close detail.

Reply:
Would you prefer I used variables subscripted with 1 or 2 as opposed to with/without nni?  Instead of `new_likelihoods` maybe `proposed_likelihoods`?

Reply:
Did our zoom conversation cover this?

Reply:
Yes, it did.  Variable names updated.

Reply:
Marvelous, thanks! Much clearer now.